src/components/HorizontalStep/HorizontalStep.module.css

```diff
@@ -4,11 +4,14 @@
 
 /**
  * Horizontal Step displays the step text and associated decorative icon.
+ * 1) Required to prevent overflowing outside container.
  */
 .horizontal-step {
```

Comment:
Another general comment: Curious how this works when there are more than a few steps and how that breaks down. I think what you have right now works fine, but just thinking about future states if needed.

Reply:
The text starts to truncate with ellipsis and eventually reaches a breakpoint where it is hidden.
The dash dividers is supposed to have a minimum width but have not heard back from design yet.
Here is an example with 5 steps and long text:

https://user-images.githubusercontent.com/86632227/167727792-57031e05-9e74-4cc5-bfcf-0b955ecbf263.mov